.github/dependabot.yml

```diff
@@ -0,0 +1,7 @@
+# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
```

Comment:
Interesting. Didn't realize you could use dependabot for the github actions themselves :)